### PR TITLE
[NAS] simplify log, and fix a bug on pdarts exporting

### PR DESCRIPTION
**Pull request by squirrelsc**

### CODE REVIEW

File: examples/nas/darts/retrain.py [hunk 1]
```diff
@@ -4,24 +4,16 @@
...
 from model import CNN
+from nni.nas.pytorch.fixed import apply_fixed_architecture
+from nni.nas.pytorch.utils import AverageMeter
 
-logger = logging.getLogger()
-
-fmt = '[%(asctime)s] %(levelname)s (%(name)s/%(threadName)s) %(message)s'
-logging.Formatter.converter = time.localtime
-formatter = logging.Formatter(fmt, '%m/%d/%Y, %I:%M:%S %p')
+logger = logging.getLogger('nni')
```
Comment (xuehui1991) [inline examples/nas/darts/retrain.py]: change another name?

File: examples/nas/darts/search.py [hunk 2]
```diff
@@ -7,20 +7,11 @@
...
-
-fmt = '[%(asctime)s] %(levelname)s (%(name)s/%(threadName)s) %(message)s'
-logging.Formatter.converter = time.localtime
-formatter = logging.Formatter(fmt, '%m/%d/%Y, %I:%M:%S %p')
-
-std_out_info = logging.StreamHandler()
-std_out_info.setFormatter(formatter)
-logger.setLevel(logging.INFO)
-logger.addHandler(std_out_info)
+logger = logging.getLogger('nni')
```
Comment (xuehui1991) [inline examples/nas/darts/search.py]: same here.

File: examples/nas/darts/retrain.py (hunk 1 above)
Comment (squirrelsc) [reply examples/nas/darts/retrain.py]: It's root logger of the code, and nni logger is initialized, so it set to nni logger here.

File: examples/nas/darts/search.py [hunk 3]
```diff
@@ -7,20 +7,11 @@
 
 import datasets
 from model import CNN
-from nni.nas.pytorch.callbacks import ArchitectureCheckpoint, LRSchedulerCallback
+from nni.nas.pytorch.callbacks import (ArchitectureCheckpoint, LRSchedulerCallback)
```
Comment (QuanluZhang) [inline examples/nas/darts/search.py]: why use brackets here?

**Pull request merged**